appinventor/appengine/src/com/google/appinventor/server/UserInfoServiceImpl.java

```diff
@@ -224,6 +227,9 @@ public void storeSharedBackpack(String backPackId, String content) {
 
   @Override
   public String deleteAccount() {
+    if (!deleteAccountAllowed) {
+      return ("");
```

Comment:
The parentheses here are allowable, but redundant.

Reply:
I program in too many languages! (Java, Rust, C, Python, Javascript, Go, Erlang) And they are all different, in some cases for no good reason!